microsoft-edge/devtools-guide-chromium/issues/index.md

```diff
@@ -43,42 +43,57 @@ Feedback in the **Issues** tool is provided by several sources, including the Ch
+-->
+
+1. In the Quick View toolbar at the bottom of DevTools, select the **Issues** tab.
+
+   If Quick View isn't displayed, select **Customize and control DevTools** (**...**), and then select **Toggle Quick View panel** (**Esc**).  If the **Quick View** toolbar doesn't have the **Issues** tab, in the **Quick View** toolbar, click the **More tools** (**+**) button, and then select **Issues**.
 
-1. Select the **Issues counter**.  The **Issues** tool opens with issues grouped into different categories.
+   The **Issues** tool groups issues into categories, such as **Accessibility**, **Performance**, **Security**, and **Other**:
 
    ![Categories of issues in the Issues tool on the demo page](./index-images/categories.png)
 
 
 ### Other ways to open the Issues tool
 
-There are several additional ways to open the **Issues** tool:
-*  Click the **More Tools** (**+**) menu in the main panel or the **Drawer**, and then select **Issues**.
+There are multiple ways to open the **Issues** tool, such as:
+
+*  In the **Activity Bar** or in **Quick View**, click the **More tools** (**+**) button, and then select **Issues**.
+*  In the **Elements** tool, in the DOM tree, find a wavy-underlined element name, and then press and hold **Shift** and click the element.  Or, right-click a wavy-underlined element, and then select **View issues**.
+<!-- legacy ui
 *  Select **Customize and control DevTools** > **More tools** > **Issues**.
-*  In the DOM tree in the **Elements** tool, press **Shift** and then click a wavy-underlined element name.  Or, open the context menu on a wavy-underlined element and then select **View issues**.
+-->
 
 
 ### Issues are automatically ordered by severity
 
-Within each category of issues, first the errors are listed, then warnings, and then tips.
+Within each category of issues, first the errors are listed, then warnings, and then tips:
 
 ![The Issues tool displays Performance issues sorted by severity](./index-images/ordered-by-severity.png)
 
+(The **Severity** dropdown list controls whether Warnings, Errors, Tips, and Info items are listed.)
+
 
 <!-- ====================================================================== -->
 ## Expand entries in the Issues tool
 
-The **Issues** tool presents additional documentation and recommended fixes to apply to each issue.  To expand an issue to get this additional information, select an issue, as follows.
+The **Issues** tool presents additional documentation and recommended fixes to apply to each issue.  To expand an issue to get this additional information:
 
-1. Open the [accessibility-testing demo page](https://microsoftedge.github.io/Demos/devtools-a11y-testing/) in a new window or tab, and then open DevTools.
+1. Open a webpage and the **Issues** tool, as described above.
 
+<!-- legacy ui
 1. Open the **Issues** tool by selecting the **Issues counter** (![Issues counter](./index-images/issues-counter-icon.png)).
+-->
```

Comment:
This comment breaks the auto-incrementing list. The next item is numbered 1, instead of 2.

Reply:
@captainbrosset - shifted html comment to col 3.  VS Code preview indicates fixed.

![image](https://github.com/MicrosoftDocs/edge-developer/assets/45407486/90570a84-4bad-4e13-b060-87df05929115)